simulation/include/pcl/simulation/camera.h

```diff
@@ -2,17 +2,17 @@
 
 #include <Eigen/Dense>
 #include <Eigen/StdVector>
-#include <boost/shared_ptr.hpp>
 
 #include <pcl/pcl_macros.h>
+#include <pcl/pcl_types.h>
 
 namespace pcl {
 namespace simulation {
 
 class PCL_EXPORTS Camera {
 public:
-  using Ptr = shared_ptr<Camera>;
-  using ConstPtr = shared_ptr<const Camera>;
+  using Ptr = pcl::shared_ptr<Camera>;
+  using ConstPtr = pcl::shared_ptr<const Camera>;
```

Comment:
In case we already enforcing the qualified/non-qualified policies we discussed, and assuming there's no namespace clash happening here, this would be a situation to let `shared_ptr` remain unqualified.